src/components/line-chart-audification/line-chart-audification.component.ts

```diff
@@ -24,14 +24,14 @@ export class LineChartAudificationComponent implements AudificationPreference, O
 
   liveText: string | null = null;
   private destroy$ = new Subject();
-  private melody?: Melody;
+  melody?: Melody;
   private domain: Date[];
   private range: number[];
   @HostBinding('attr.tabindex') private readonly tabindex = 0;
   private readOutTimeoutId: number | null = null;
 
   constructor(
-    @Inject('host') private host: LineChartComponent,
+    @Inject('host') public host: LineChartComponent,
     private zone: NgZone,
   ) {
```

Comment:
We try to avoid making class items public purely for the sake of testing. Is there another way to test this component without accessing private instance variables and methods? 

Could you create an instance of the host lineChartComponent and supply that instance using `useValue` in the providers array? And then test the readOut method through simulating keyUp and keyDown events?

Reply:
That makes sense. I was able to make `host` private using `useValue`.

For the `readOut` method, I moved it out to `ScreenReaderComponent` and made it public in #34.